pkg/combination/combination_test.go

```diff
@@ -35,25 +35,46 @@ var combinationTests = []struct {
 			err:          nil,
 		},
 	},
+	{
+		name:  "standard set of long args",
+		input: testdata.LongCombinationInput,
+		expected: expected{
+			combinations: testdata.LongCombinationOutput,
+			err:          nil,
+		},
+	},
```

Comment:
I think we should avoid trying to tack on extra test cases to this global testing spec, and instead define it in-line using table-driven tests.